shell/shared/demo.js

```diff
@@ -84,7 +84,21 @@ if (Meteor.isServer) {
         // Log them in on this connection.
         return Accounts._loginMethod(this, "createDemoUser", arguments,
             "demo", function () { return { userId: userId }; });
+      },
+
+      getAppNameFromAppId: function(packageId) {
```

Comment:
This should be a Meteor.publish() instead of a method, since it's just providing access to some of the database. That way:
- If the app name changes the page will update (not really important in this case, but a good habit).
- You can block rendering while you wait for the subscription.
- You don't have to use a session variable to store the result.